docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1,11 +1,8 @@
-### Style guide for Flutter repo
+# Style guide for Flutter repo
 
 ## Summary
 
-Optimize for readability. Write detailed documentation.
-Make error messages useful.
-Never use timeouts or timers.
-Avoid `is`, `print`, `part of`, `extension` and `_`.
+Optimize for readability. Write detailed documentation. Make error messages useful. Never use timeouts or timers. Avoid `is`, `print`, `part of`, `extension` and `_`.
```

Comment:
After a whitespace change, there wouldn't be any diffs here!

Reply:
(small side-note: the [style guide](https://github.com/flutter/flutter/blob/98fb56fc057753884454ddd0c01cf7cb643768f9/docs/contributing/Style-guide-for-Flutter-repo.md) doesn't use a bullet list currently, those minus signs are to show the diffs)

Reply:
Yeah, I misread the diff in the exact same way originally. The status quo is a paragraph. The current revision of the PR just unwraps it onto one source line, and @nate-thegrate's comment above was pointing out that that can be undone so that this diff hunk disappears from the diff.

The original revision of the PR actually proposed turning the paragraph into a bulleted list. My reaction then was that I kind of like the effect of the paragraph, but either way seems fine and mostly I'm inclined to keep it as is rather than make stylistic changes:
https://github.com/flutter/flutter/pull/150167#discussion_r1648429267